src/RobotInterface/include/BipedalLocomotion/RobotInterface/ISensorBridge.h

```diff
@@ -75,13 +72,19 @@ struct SensorBridgeMetaData
  */
 class ISensorBridge
 {
+protected:
+    using Vector12d = Eigen::Matrix<double, 12, 1>;
+    using Vector6d = Eigen::Matrix<double, 6, 1>;
```

Comment:
Here you changed the accessibility of these aliases. Before they were public, now they are not accessible by the user. if these are used only in the implementation, I would move them in a cpp file.

Reply:
I put it there because before where not in a namespace so everyone including the files had Vector aliases.  However it’s nice to have it in the h because they simplify the declaration of the methods.

a possible solution is to include them in math component so everyone can use them

Reply:
As you prefer. Just to express my point a little better, either set them as ``public`` or hide them completely. It would be weird to declare public methods using a private alias.

Reply:
Ok I put them public but inside ISensorBridge class :smile_cat:

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/226/commits/7ad35bcf0091a6a3271fe7d5a218400d39582c53